src/Planners/tests/UnicycleTrajectoryGeneratorTest.cpp

```diff
@@ -0,0 +1,98 @@
+/**
+ * @file UnicycleTrajectoryGeneratorTest.cpp
+ * @copyright 2020 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the BSD-3-Clause license.
+ */
+#include <catch2/catch_test_macros.hpp>
+
+#include <manif/manif.h>
+
+#include <iDynTree/Model.h>
+#include <iDynTree/ModelLoader.h>
+
+#include <BipedalLocomotion/ParametersHandler/StdImplementation.h>
+#include <BipedalLocomotion/Planners/UnicycleTrajectoryGenerator.h>
+#include <BipedalLocomotion/TextLogging/Logger.h>
+
+#include <fstream>
+
+#include <FolderPath.h>
+
+using namespace BipedalLocomotion::Planners;
+using namespace BipedalLocomotion::ParametersHandler;
+
+void saveData(const UnicycleTrajectoryGeneratorOutput& output, const std::string& filename)
+{
+    std::ofstream file(filename);
+    if (!file.is_open())
+    {
+        BipedalLocomotion::log()->error("[saveData] Unable to open file {} for writing.", filename);
+        return;
+    }
+
+    file << output.contactPhaseList.toString() << std::endl;
+
+    file.close();
```

Comment:
You're right, I took inspiration from the old tests. Indeed, I think there is no need to save anything for this simple CI test. So, I might simply remove this part. Also because I have written [a more complete example](https://github.com/ami-iit/bipedal-locomotion-framework/blob/3e400dfa0f8652bda08a9f825c670305289c4008/examples/UnicycleTrajectoryGenerator/main.cpp), which is provided by this PR, where it makes sense to save results, and in this case I am doing with `matioCpp`.

Reply:
done with https://github.com/ami-iit/bipedal-locomotion-framework/pull/845/commits/44f6bc52d51e7e996e5c7464b05cd489a86dd465